core/src/retransmit_stage.rs

```diff
@@ -42,7 +41,9 @@ use {
 };
 
 const MAX_DUPLICATE_COUNT: usize = 2;
-const DEFAULT_LRU_SIZE: usize = 1 << 20;
+const DEDUPER_FALSE_POSITIVE_RATE: f64 = 0.001;
+const DEDUPER_NUM_BITS: u64 = 637_534_199; // 76MB
```

Comment:
so 152MB for the full patch set (shred-fetch + retransmit)?

Reply:
yes